src/lib.rs

```diff
@@ -993,6 +1001,48 @@ where
     }
 }
 
+#[cfg(feature = "std")]
+impl<T> AsDecimal for Ratio<T> where
+    T: fmt::Display
+        + Clone
+        + Eq
+        + FromPrimitive
+        + Integer
+        + Pow<usize, Output = T>
```

Comment:
With `Clone + Integer`, we can just use `num_traits::pow`.

Reply:
Used `traits::pow` instead.